docs/Makefile

```diff
@@ -0,0 +1,6 @@
+GOMD2MAN = go-md2man
+
+docs: $(patsubst %.md,%.1,$(wildcard *.md))
+
+%.1: %.md
+	$(GOMD2MAN) -in $^ -out $@
```

Comment:
Should add something to the install block, but we can get that on a future patch.